sources/packages/web/src/components/common/NoticeOfAssessmentFormView.vue

```diff
@@ -91,6 +91,9 @@ export default defineComponent({
     ) => {
       return !!applicationId && !!loadNOA && !!processing;
     },
+    setHideView: (value: boolean) => {
```

Comment:
I see the point of trying to handle an error scenario but IMO we would be just changing the error. Not loading the data on the form or not displaying the form will not inform the user properly about what happened.
I would prefer to have this type of error handled in a global way, for instance,
- detecting the error at the API level calls,
- detecting the error at the UI page level and maybe creating a common container and helper methods, composables, etc.

Reply:
ya, I agree, I was thinking when we implement the 404 forbidden pages, maybe we can detect the error in the API service level on the UI side (like a global handler) and redirect to 404 forbidden pages. and we can remove these checks

Reply:
as discussed removed